ipfx/dataset/ephys_data_set.py

```diff
@@ -70,6 +70,22 @@ def sweep_table(self) -> pd.DataFrame:
             self._sweep_table = pd.DataFrame(sweeps)
         return self._sweep_table
 
+    @property
```

Comment:
this is different from updating the documentation. I suggest to create a separate issue.

Reply:
I don't agree. This change is minor, purely additive, & allows us to rescue the inline code samples in the documentation.